### DISCUSSION TITLE
请问有办法获取当前界面吗，比如是主界面，或是房间中、游戏中

### DISCUSSION THREAD

**Discussion by XilyFeAAAA** [FrozenFish259/Goose_Goose_Duck_Hack#139] [General 聊天区]
我需要在游戏开始时初始化一些参数，可以知道什么时候进入游戏吗

**Comment by FrozenFish259** [comment]
我现在只是通过读取玩家数据来判断，后期我会替换成别的方法

**Reply by XilyFeAAAA** [reply]
好的，谢谢了

**Reply by FrozenFish259** [reply]
LobbySceneHandler

**Reply by FrozenFish259** [reply]
![image](https://user-images.githubusercontent.com/49237108/212875887-9a39a2c6-bed9-45e9-b913-5230cf9a49da.png)

**Reply by FrozenFish259** [reply]
地图信息之类的也有,还有房间号等等

**Reply by XilyFeAAAA** [reply]
谢谢，很有帮助(*^▽^*)
